docs/design-documents/tools/cmake.md

```diff
@@ -32,20 +32,19 @@ There are numerous CMake files in the Mbed OS repository tree:
 * A `CMakeLists.txt` entry point in the Mbed OS root, describing the top level build specification for the Mbed OS source tree.
 * `CMakeLists.txt` entry points in each Mbed OS module subdirectory, describing the build specification for a module or component
 
-A number of CMake scripts are contained in the `mbed-os/cmake` directory:
-* `toolchain.cmake` - selects the toolchain script from the `cmake/toolchains` directory, based on the value of the `MBED_TOOLCHAIN` variable
-* `profile.cmake` - selects the profile script from the `cmake/profiles` directory, based on the value of the `MBED_PROFILE` variable
+A number of CMake scripts are contained in the `mbed-os/tools/cmake` directory:
 * `core.cmake` - selects the core script from the `cmake/cores` directory, based on the value of the `MBED_CPU_CORE` variable
+* `profile.cmake` - selects the profile script from the `cmake/profiles` directory, based on the value of the `MBED_PROFILE` variable
```

Comment:
I only removed app.cmake and then realized we should organize it the same way as it is in the tree (alphabetically).